helm/servicex/values.yaml

```diff
@@ -15,8 +15,8 @@ app:
     tls:
       clusterIssuer: null
       enabled: false
-      secretName: '{{.Release.Name}}-app-tls'
-  logLevel: WARNING
+      secretName: "{{.Release.Name}}-app-tls"
+  logLevel: INFO
```

Comment:
We shouldn't be changing log levels in the code.  You can set the levels using the LOG_LEVEL env var for the servicex container or by setting flask to debug mode.

Reply:
but this is not in the code. This is simply a change of the default in values.yaml?

Reply:
Or in the default deployment.  I believe the convention is to use WARNING/WARN as the default to avoid cluttering up the logs with extraneous messages.  We should stick to the convention and then increase it in our testing instances as needed.

Reply:
Well, the default values can be overridden in the environment of the container.  If we need more information, why not update the values.yaml in our flux deployments to set the log level to DEBUG?  Also, it sounds like we should also be enabling flask debugging as well in our deployments.  This is something that's relatively easy to do without having to change the default.yaml for the chart.